tests/shared/UserLogonStateChangeTestDefs.js

```diff
@@ -185,7 +192,7 @@ gpii.tests.userLogonHandling.testDefs = [{
     }, {
         // wait for debounce
         func: "setTimeout",
-        args: [ "{tests}.events.timeoutComplete.fire", 6000 ]
+        args: [ "{tests}.events.timeoutComplete.fire", 3500 ]
```

Comment:
I think we should also change the wait time for the expected debounce cases. From the current 1000ms to maybe 500ms or 100ms.

Reply:
And while we are in this code, could we change the comments from "wait for debounce" and "wait, but not long enough for debounce" to something more descriptive? These comments are sort of the opposite of what we mean. "wait for debounce" is really "wait for the debounce period to pass so that our request is not debounced". And "wait, but not long enough for debounce" is really "send 2 requests within the debounce period so that we trigger the debounce logic".